packages/core/src/App/Containers/NotificationsDialog/notifications-dialog.jsx

```diff
@@ -54,6 +56,14 @@ const NotificationsList = ({ notifications, toggleDialog }) => {
                             />
                         )}
                         {item.header}
+                        {item.has_close_dialog_icon && (
+                            <Icon
+                                className='notifications-item__title-close-icon'
+                                icon='IcCloseCircle'
+                                onClick={() => removeNotificationByKey(item)}
+                                custom_color='grey'
+                            />
+                        )}
```

Comment:
Added close icon in notification dialog as clarified by design team. For a notification to enable this close icon, they must pass in `has_close_dialog_icon: true` in the object when calling `addNotificationMessage`
<img width="343" alt="Screenshot 2023-02-08 at 3 50 27 PM" src="https://user-images.githubusercontent.com/103016120/217480914-598221e9-b625-404f-9f55-b8f1db89ec6e.png">